pkg/api/defaults.go

```diff
@@ -651,7 +647,10 @@ func (p *Properties) setAgentProfileDefaults(isUpgrade, isScale bool) {
 			}
 
 			if profile.SinglePlacementGroup == nil {
-				if p.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == StandardLoadBalancerSku {
+				if p.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == "" {
+					p.OrchestratorProfile.KubernetesConfig.LoadBalancerSku = p.GetLoadBalancerDefault()
```

Comment:
The LB SKU is cluster default, should not be set in setAgentProfileDefaults
It think it's safe to assume LB sku is not "" at this step, otherwise we should fix in the orchestrator default

Reply:
actually it's set at L347 right?

Reply:
no, it is not like that. setAgentPoolDefaults() called before setOrchestratorDefaults(), where LB sku default sets. It is possible that LB sku is empty at setAgentPoolDefault().